Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -311,6 +344,18 @@ def audit_installed
   def relative_glob(dir, pattern)
     File.directory?(dir) ? Dir.chdir(dir) { Dir[pattern] } : []
   end
+
+  def cpuid_instruction?(file, objdump = "objdump")
+    has_cpuid_instruction = false
+    Utils.popen_read(objdump, "-d", file) do |io|
+      until io.eof?
+        has_cpuid_instruction = io.readline.include? "cpuid"
+        break if has_cpuid_instruction
+      end
+    end
+
+    has_cpuid_instruction
+  end
```

Comment:
Not sure if this is the best way to do this, but the idea here is that I don't want to wait for `objdump` to finish dumping its output (which could take a while) when it has already produced a line containing a `cpuid` instruction.

This makes `brew audit --strict openblas` run almost twice as fast compared to the naive approach of
```ruby
Utils.popen_read(objdump, "-d", file).include? "cpuid"
```